### ISSUE TITLE
Pathfinder example references pymcx instead of pymc_experimental

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc-examples#503]
We decided to keep the long name for the library, so the example should be updated.

CC @twiecki

**Comment by 2022H1030042G** [comment]
Hey, I would like to work on this issue, is it available?

**Comment by ricardoV94** [comment]
Yes, feel free to take it

**Comment by 2022H1030042G** [comment]
I am new to this field, can you please guide me on this?

**Comment by fonnesbeck** [comment]
You will want to:

1. Fork the pymc-examples repo
2. Create a branch to work on the issue
3. From the branch, replace occurrences of `pymcx`  with `pymc_experimental`
4. Push the branch to GitHub
5. Create a pull request to this repo from your branch

Hope that helps. Feel free to ask if you have additional questions.

**Comment by reshamas** [comment]
In case you need to set up your virtual environment, there are some instructions here:  
https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/environment_setup.html

The virtual environment will be helpful for running the notebook after the changes have been made to see that they work, error free

**Comment by 2022H1030042G** [comment]
Where is this "pymcx"?, I am unable to find it,

**Comment by fonnesbeck** [comment]
It doesn't exist, which is why we have to change it.

**Comment by twiecki** [comment]
Specifically, we renamed pymcx to `pymc-experimental`, which is what you have to do here. https://github.com/pymc-devs/pymc-experimental/

**Comment by drbenvincent** [comment]
This page is a useful guide https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/docstring_tutorial.html 

And the style guide is useful to consult if you run into any issues https://docs.pymc.io/en/latest/contributing/jupyter_style.html

**Comment by reshamas** [comment]
code:  
```python
import arviz as az
import numpy as np
import pymc as pm
import pymc_experimental as pmx
import pytensor
import pytensor.tensor as pt

print(f"Running on PyMC v{pm.__version__}")
```

```python
with model:
    idata = pmx.fit(method="pathfinder")
```

Error:
```bash
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
File ~/miniforge3/envs/pymc-dev/lib/python3.11/site-packages/pymc_experimental/inference/fit.py:34, in fit(method, **kwargs)
     33 try:
---> 34     from pymc_experimental.inference.pathfinder import fit_pathfinder
     35 except ImportError as exc:

File ~/miniforge3/envs/pymc-dev/lib/python3.11/site-packages/pymc_experimental/inference/pathfinder.py:21
     20 import arviz as az
---> 21 import blackjax
     22 import jax

ModuleNotFoundError: No module named 'blackjax'

The above exception was the direct cause of the following exception:

RuntimeError                              Traceback (most recent call last)
Cell In[3], line 2
      1 with model:
----> 2     idata = pmx.fit(method="pathfinder")

File ~/miniforge3/envs/pymc-dev/lib/python3.11/site-packages/pymc_experimental/inference/fit.py:36, in fit(method, **kwargs)
     34     from pymc_experimental.inference.pathfinder import fit_pathfinder
     35 except ImportError as exc:
---> 36     raise RuntimeError("Need BlackJAX to use `pathfinder`") from exc
     37 return fit_pathfinder(**kwargs)

RuntimeError: Need BlackJAX to use `pathfinder`
```

**Comment by reshamas** [comment]
jax installation:  
https://github.com/google/jax#installation

blackjax installation:
https://pypi.org/project/blackjax/

**Comment by twiecki** [comment]
Closed by #521.
